polaris-react/src/components/TextField/TextField.scss

```diff
@@ -315,6 +366,11 @@ $spinner-icon-size: 12px;
   width: 22px;
   cursor: pointer;
 
+  #{$se23} & {
+    justify-content: center;
+    margin: var(--p-space-1);
+  }
+
```

Comment:
This change and below relate to the "Spinners" aka the stepper

| Before | After |
|-|-|
| <img width="521" alt="Screenshot 2023-06-02 at 4 14 34 PM" src="https://github.com/Shopify/polaris/assets/20652326/d477d4fd-4622-4ede-8e75-e75f5d2f1e7f"> | <img width="534" alt="Screenshot 2023-06-02 at 4 14 23 PM" src="https://github.com/Shopify/polaris/assets/20652326/fd5c3ebe-e0b9-40ab-ba5a-e43646d4fa99"> |